Library/Homebrew/test/cask/artifact/shared_examples/uninstall_zap.rb

```diff
@@ -294,13 +292,13 @@
   [:delete, :trash].each do |directive|
     next if directive == :trash && ENV["HOMEBREW_TESTS_COVERAGE"].nil?
 
-    context "using :#{directive}" do
+    context "when using :#{directive}" do
       let(:dir) { TEST_TMPDIR }
       let(:absolute_path) { Pathname.new("#{dir}/absolute_path") }
       let(:path_with_tilde) { Pathname.new("#{dir}/path_with_tilde") }
-      let(:glob_path1) { Pathname.new("#{dir}/glob_path1") }
-      let(:glob_path2) { Pathname.new("#{dir}/glob_path2") }
-      let(:paths) { [absolute_path, path_with_tilde, glob_path1, glob_path2] }
+      let(:glob_path) { Pathname.new("#{dir}/glob_path") }
+      let(:glob_path_) { Pathname.new("#{dir}/glob_path_") }
+      let(:paths) { [absolute_path, path_with_tilde, glob_path, glob_path_] }
```

Comment:
This one doesn't make that much sense to me.

Reply:
Open to alternatives, just trying to obey the letter and not the spirit of the cop. 😈

Reply:
`glob_path_alt` ?

Reply:
> Open to alternatives, just trying to obey the letter and not the spirit of the cop. 😈

🤣 

> `glob_path_alt` ?

That's a bit better. I'm fine with that.